src/components/Nav.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom'
+import NotFound from './notFound';
+// import NotFound2 from './notFound2';
+// import NotFound3 from './notFound3';
+import Home from './Home';
+
+const Nav: React.FC = () => {
+    return (
+        <div>
+            <Routes>
+                <Route path="/" element={<Home />}></Route>
+                <Route path="/notfound1" element={<NotFound />}></Route>
```

Comment:
could each of these paths be `/notfound`? Is that allowed?

Reply:
I thought I had already made that change but I will go back and check.